nginx-proxy/nginx.tmpl

```diff
@@ -84,6 +84,9 @@
 	
 {{ end }}
 
+js_include imgpath.js;
```

Comment:
can this go inside the `{{ define "thumbor" }}` section?

One thing I was trying to do was to make any modification of the upstream nginx.tmpl isolated, as much as possible... It's not exactly super-clear or easy, I admit. But the idea was that if we define things above, and then just inject a single line, then it would be easier to maintain...

Reply:
I'm afraid no. js_include can only be used outside the `server` section